### ISSUE TITLE
[Accessibility] Name Property does not exist for Description Group Control.

### ISSUE THREAD

**Issue by asthanarht** [tinymce/tinymce#5189]
Create Area box and attached minimal tinymce area 
![378b5e2c-1a5f-4157-957f-ad1f326a8ac0](https://user-images.githubusercontent.com/4268434/65793520-e04c3200-e11a-11e9-9af7-7f9ee5552319.png)

Actual:
Name Property does not exist for Description Group Control.

There is no way to assign Aria-Label in Tiny MCE control.

**Comment by lnewson** [comment]
This isn't clear enough about what you want to do or what your configuration is. Please include a [fiddle](http://fiddle.tinymce.com/) and provide more information about what your issue is.

**Comment by emolayi** [comment]
I don't know if this is the same issue, but I have failing accessibility tests that give me the following:
```
       1) aria-allowed-attr: Elements must only use allowed ARIA attributes (critical)
           https://dequeuniversity.com/rules/axe/3.3/aria-allowed-attr?application=axeAPI
           The following 1 node violate this rule:
           
               Selector: #mceu_41-0
               HTML: <div role="button" class="mce-path-item mce-last" data-index="0" tabindex="-1" id="mceu_41-0" aria-level="1">p</div>
               Fix any of the following:
               - ARIA attribute is not allowed: aria-level="1"
```

I am using the tinymce-rails 4.9.6 gem, rails 5.2.3, and ruby 2.6.5.

**Comment by lnewson** [comment]
Closing this due to lack of additional information. If you can provide the additional information mentioned above, then we're more than happy to reopen this issue.

@emolayi thats a separate issue, but it should no longer be an issue in TinyMCE 5 and unfortunately we're only providing critical fixes for community version of TinyMCE 4 now.
